src/reporting/Interfaces.sol

```diff
@@ -34,12 +33,22 @@ contract IMarket is Typed {
     function isDoneWithAllReporters() constant public returns (bool);
     function isDoneWithLimitedReporters() constant public returns (bool);
     function getReportingTokenOrZeroByPayoutDistributionHash(bytes32) constant public returns (ReportingToken);
+    function getFinalizationTime() constant public returns (uint);
+    function getDenominationToken() constant public returns (ERC20);
+    function getPayoutDenominator() constant public returns (int256);
+    function getShareToken(uint8) constant public returns (IShareToken);
+    function getCompleteSetCostInAttotokens() constant public returns(int256);
+    function getMarketCreatorSettlementFeeInAttoethPerEth() constant public returns (int256);
+    function getCreator() constant public returns (address);
+    function shouldCollectReportingFees() constant public returns (bool);
 }
 
 
 contract IShareToken is Typed {
     function initialize(IMarket _market, uint8 _outcome) public returns (bool);
     function getMarket() constant returns (IMarket);
+    function balanceOf(address) constant returns (int256);
```

Comment:
Rather than adding `balanceOf` here, recommend making `IShareToken` derive from `ERC20` which will bring in all of the token stuff.